resources/qml/Cura.qml

```diff
@@ -56,6 +50,22 @@ UM.MainWindow
         color: UM.Theme.getColor("window_disabled_background")
         opacity: 0.7
         z: stageMenu.z + 1
+
+        MouseArea
+        {
+            // Prevent all mouse events from passing through.
+            enabled: parent.visible
+            anchors.fill: parent
+            hoverEnabled: true
+            acceptedButtons: Qt.LeftButton | Qt.RightButton
```

Comment:
Can still use the middle mouse button to strafe the camera?

Reply:
Fixed here: https://github.com/Ultimaker/Cura/pull/5449/commits/745d9e61161490edacb46eb7a6d25abe6e922ada